From cead646dfc59be397fefc2df29f39207d4edae1e Mon Sep 17 00:00:00 2001
From: Jerry Jelinek <jerry.jelinek@joyent.com>
Date: Thu, 10 Jan 2019 17:08:58 +0000
Subject: [PATCH] MANTA-4053 DMU tx throttling of XLogFileInit() during
 autovacuum induces extreme outliers Reviewed by: Kody Kantor
 <kody.kantor@joyent.com> Approved by: Kody Kantor <kody.kantor@joyent.com>

---
 deps/postgresql96                        | 2 +-
 etc/9.6/postgresql.manta.coal.conf       | 1 +
 etc/9.6/postgresql.manta.lab.conf        | 1 +
 etc/9.6/postgresql.manta.production.conf | 1 +
 4 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/deps/postgresql96 b/deps/postgresql96
index 5f50a8f..fda6203 160000
--- a/deps/postgresql96
+++ b/deps/postgresql96
@@ -1 +1 @@
-Subproject commit 5f50a8f2b09c0e35d72d878c3d498a221aeeb4b8
+Subproject commit fda6203fca5c8a4bd3dda2dc725cf02134e8725e
diff --git a/etc/9.6/postgresql.manta.coal.conf b/etc/9.6/postgresql.manta.coal.conf
index 8fd2fc8..7383c48 100644
--- a/etc/9.6/postgresql.manta.coal.conf
+++ b/etc/9.6/postgresql.manta.coal.conf
@@ -653,3 +653,4 @@ default_text_search_config = 'pg_catalog.english'
 
 # Add settings for extensions here
 wal_recycle = off
+wal_init_zero = off
diff --git a/etc/9.6/postgresql.manta.lab.conf b/etc/9.6/postgresql.manta.lab.conf
index af89a11..b00e847 100644
--- a/etc/9.6/postgresql.manta.lab.conf
+++ b/etc/9.6/postgresql.manta.lab.conf
@@ -653,3 +653,4 @@ default_text_search_config = 'pg_catalog.english'
 
 # Add settings for extensions here
 wal_recycle = off
+wal_init_zero = off
diff --git a/etc/9.6/postgresql.manta.production.conf b/etc/9.6/postgresql.manta.production.conf
index fac0e2e..9e591f0 100644
--- a/etc/9.6/postgresql.manta.production.conf
+++ b/etc/9.6/postgresql.manta.production.conf
@@ -654,3 +654,4 @@ default_text_search_config = 'pg_catalog.english'
 
 # Add settings for extensions here
 wal_recycle = off
+wal_init_zero = off
-- 
2.21.0

